hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenRelListCalculator.java

```diff
@@ -234,21 +234,25 @@ private TokenRelation getInPriorityOrder(
     @NonNull
     private TokenRelPointerUpdateResult updatePointersSurroundingTargetTokenRel(
             @Nullable TokenRelation prevTokenRel, @Nullable final TokenRelation nextTokenRel) {
-        final var prevTokenRelTokenNum = prevTokenRel != null ? prevTokenRel.tokenNumber() : -1;
-        final var nextTokenRelTokenNum = nextTokenRel != null ? nextTokenRel.tokenNumber() : -1;
+        final var prevTokenRelTokenId = prevTokenRel != null
+                ? prevTokenRel.tokenId()
+                : TokenID.newBuilder().tokenNum(-1L).build();
+        final var nextTokenRelTokenId = nextTokenRel != null
+                ? nextTokenRel.tokenId()
+                : TokenID.newBuilder().tokenNum(-1L).build();
```

Comment:
same here , return null and instead of using -1 everywhere we will see if prevToken and nextToken are null

Reply:
same as above.